lifecycle/cli.go

```diff
@@ -379,6 +389,37 @@ func releaseBundleImport(c *components.Context) error {
 	return commands.Exec(importCmd)
 }
 
+func annotate(c *components.Context) error {
+	if show, err := pluginsCommon.ShowCmdHelpIfNeeded(c, c.Arguments); show || err != nil {
+		return err
+	}
+
+	if len(c.Arguments) != 2 {
+		return pluginsCommon.WrongNumberOfArgumentsHandler(c)
+	}
+
+	rtDetails, err := createLifecycleDetailsByFlags(c)
+	if err != nil {
+		return err
+	}
+	annotateCmd := lifecycle.NewReleaseBundleAnnotateCommand()
+
+	project := c.GetStringFlagValue("project")
```

Comment:
as suggested by @fluxxBot please use `pluginsCommon.GetProject(c)` otherwise reading project via environment variable(JFROG_CLI_BUILD_PROJECT) will be missed.